recipes/notion-client/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "notion-client" %}
+{% set version = "0.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/notion-client-{{ version }}.tar.gz
+  sha256: 58519c4340f99c2b7651a1d671f07411052fb46786fcb41df71c60e6b1b144ac
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+  run:
+    - httpx >=0.15.0
+    - python >=3.6,<4.0
+
+test:
+  imports:
+    - notion_client
+    - tests
```

Comment:
If this works, this means that this package installs a global-namespaced `tests` package. That should not be part of it.

Reply:
Yikes, I missed that. I can confirm that the wheel does indeed do this:
```
unzip ../notion_client-0.7.1-py2.py3-none-any.whl
Archive:  ../notion_client-0.7.1-py2.py3-none-any.whl
  inflating: notion_client/__init__.py  
  inflating: notion_client/api_endpoints.py  
  inflating: notion_client/client.py  
  inflating: notion_client/errors.py  
  inflating: notion_client/helpers.py  
  inflating: notion_client/logging.py  
  inflating: notion_client/py.typed  
  inflating: notion_client/typing.py  
  inflating: tests/__init__.py       
  inflating: tests/conftest.py       
  inflating: tests/test_client.py    
  inflating: tests/test_helpers.py   
  inflating: notion_client-0.7.1.dist-info/LICENSE  
  inflating: notion_client-0.7.1.dist-info/METADATA  
  inflating: notion_client-0.7.1.dist-info/WHEEL  
  inflating: notion_client-0.7.1.dist-info/top_level.txt  
  inflating: notion_client-0.7.1.dist-info/RECORD  
```

Reply:
It has been 0 days since we had a [package that](https://github.com/stac-utils/pystac-client/pull/122) that installs a `tests` module. Our record is 0 days.

Reply:
😆 🙄  It seems to be an eternal battle between the "challenge" of Python packaging and developers completely misconfiguring the packaging.